tests/pylorax/test_buildstamp.py

```diff
@@ -16,8 +16,12 @@ def setUp(self):
 
     def test_write_produces_file_with_expected_content(self):
         out_file = io.StringIO()
+        # in Python 2 write accepts unicode, not string
```

Comment:
This is a quirk of io.StringIO, not Python2. io.StringIO was designed for py3. https://docs.python.org/2.7/library/io.html

Reply:
Fixed, I meant to document this properly but was experimenting with io.StringIO vs (c)StringIO.StringIO and forgot about it.